### PR TITLE
Update our readmes and build and push our regularly built dev tags

### PR DESCRIPTION
**Pull request by condnsdmatters**
weekly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,11 +81,21 @@ $ pip install -e ".[dev]"
 $ pre-commit install
 ```
 
+On **Windows**, prebuilt docker images are hosted on our DockerHub.
```
Comment (heiner) [inline README.md:84]: Perhaps "For Windows"? Arguably the availability of Docker images isn't directly related to the host OS?

File: .github/workflows/build_docker.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ jobs:
     - uses: actions/checkout@v2
       with:
         submodules: true
-    - name: ${{ format('Build {0} image (& Publish - Release Only)', matrix.distro) }}
+    - name: ${{ format('Build & Publish `dev` {0} image', matrix.distro) }}
       uses: docker/build-push-action@v1
       with:
         username: edran
```
Comment (heiner) [inline .github/workflows/build_docker.yml:36]: Maybe an opportunity to update the username here?

File: README.md (hunk 1 above)
Comment (condnsdmatters) [reply README.md:84]:
```suggestion
For **Windows**, prebuilt docker images are hosted on our DockerHub. 
```

**Comment by heiner** [comment]
Hey @condnsdmatters, what's the status of this PR?

**Comment by condnsdmatters** [comment]
Superceded by #345.  For some reason I cant create secrets for this repo, so I think that if it aint broke, fix it not.

**Pull request closed without merging**